server/master/src/main/java/org/apache/accumulo/master/FateServiceHandler.java

```diff
@@ -232,7 +232,7 @@ public String invalidMessage(String argument) {
           propertiesToSet.put(entry.getKey(), entry.getValue());
         }
 
-        master.fate.seedTransaction(opid, new TraceRepo<>(new CloneTable(c.getPrincipal(), srcTableId, tableName, propertiesToSet, propertiesToExclude)),
+        master.fate.seedTransaction(opid, new TraceRepo<>(new CloneTable(c.getPrincipal(), srcTableId, tableName, propertiesToSet, propertiesToExclude, namespaceId)),
```

Comment:
This is a style comment, feel free to ignore. For the recent FATE changes I made, when I added a namespaceId parameter I added it before the tableId.